app/controllers/application_controller/current_user.rb

```diff
@@ -12,14 +16,12 @@ def clear_current_user
     session[:userid]    = nil
     session[:group]     = nil
   end
-  protected :clear_current_user
```

Comment:
Not for this PR, but think this method could go away entirely and one could just use `current_user = nil`, with some minor tweaks to `current_user=`